modules/core/src/main/java/org/apache/ignite/internal/processors/affinity/GridAffinityAssignmentCache.java

```diff
@@ -724,27 +722,13 @@ private void awaitTopologyVersion(AffinityTopologyVersion topVer) {
     }
 
     /**
-     * @param aff Added affinity assignment.
+     * Cleaning the affinity history.
      */
-    private void onHistoryAdded(GridAffinityAssignment aff) {
-        int fullSize = fullHistSize.incrementAndGet();
-
-        int size;
-
-        if (aff.clientEventChange())
-            size = histSize.get();
-        else
-            size = histSize.incrementAndGet();
-
-        int rmvCnt = size - MAX_HIST_SIZE;
-
-        if (rmvCnt <= 0) {
-            if (fullSize > MAX_HIST_SIZE * 2)
-                rmvCnt = MAX_HIST_SIZE;
-        }
-
-        if (rmvCnt > 0) {
+    private void onHistoryAdded() {
+        if (fullHistSize.incrementAndGet() > MAX_HIST_SIZE) {
             Iterator<HistoryAffinityAssignment> it = affCache.values().iterator();
```

Comment:
According to our code style, there should be blank lines between code lines within the same { } block.

Reply:
done